.gitignore

```diff
@@ -141,3 +141,16 @@ secrets
 
 integration_tests/scripts/set_env.sh
 
+
+# Edited by wzya
+integration_tests/scripts/.terraform/providers/registry.terraform.io/hashicorp/aws/4.22.0/darwin_arm64/terraform-provider-aws_v4.22.0_x5
+integration_tests/scripts/splunk-connect-for-snmp.tgz
+integration_tests/scripts/terraform.tfstate
+integration_tests/scripts/terraform.tfstate.backup
+integration_tests/scripts/.ipynb_checkpoints
+integration_tests/scripts/.terraform.lock.hcl
+integration_tests/scripts/inventory.yaml
+integration_tests/scripts/main.tf
+integration_tests/scripts/Untitled.ipynb
```

Comment:
I think `Untitled.ipynb` was here because I once created jupyter here, this is not the file that would be created ever. Same for `.ipynb_checkpoints`